### DISCUSSION TITLE
Would Advanced AWS Lambda API Work with Next.js pages/api ?

### DISCUSSION THREAD

**Discussion by bahdcoder** [vercel/vercel#5947] [Help] [answered]
Hello. Just went through [the advanced lambda api docs](https://vercel.com/docs/runtimes#advanced-usage/advanced-node-js-usage/aws-lambda-api) and I'm wondering if it will work with the `pages/api` in Next.js?

If yes, how to set it up please?

**Comment by styfle** [comment, marked as answer]
The [AWS Lambda API](https://vercel.com/docs/runtimes#advanced-usage/advanced-node-js-usage/aws-lambda-api) is for existing Serverless Functions you wish to deploy to Vercel but do not want to change the API.

It will not work with Next.js [API Routes](https://nextjs.org/docs/api-routes/introduction).
